include/wavm/WAVMWasmModule.h

```diff
@@ -114,6 +117,9 @@ class WAVMWasmModule final
       uint32_t stackTop,
       WAVM::Runtime::ContextRuntimeData* contextRuntimeData);
 
+    std::unordered_map<int32_t, uint32_t> chainedThreads;
```

Comment:
This is a map of wasm offsets to message IDs, so we can't use an array unfortunately.